### PR TITLE
login via redirect

### PR DESCRIPTION
**Pull request by obmelvin**

### CODE REVIEW

File: src/auth/LoginForm.tsx [hunk 1]
```diff
@@ -187,6 +199,10 @@ const LoginForm = ({
         <JSForm {...formProps} _internalId={_internalId} />
       </div>
     );
+  } else if (apexForm) {
+    // If logged in and have finished onboarding questions for an apex form, then we need to redirect back to application
+    window.location.href = 'http://localhost:3000';
```
Comment (obmelvin) [inline src/auth/LoginForm.tsx:204]: should this be set in the dashboard or via URL param when redirected to the apex form?

File: src/auth/LoginForm.tsx [hunk 2]
```diff
@@ -187,6 +199,10 @@ const LoginForm = ({
         <JSForm {...formProps} _internalId={_internalId} />
       </div>
     );
+  } else if (apexForm) {
```
Comment (obmelvin) [inline src/auth/LoginForm.tsx:202]: for my POC, I passed this prop in to LoginForm in hosted forms. for the full implementation we only want to pass it in for forms that are actually the apex form

do you think we identify that based on URL (lack of slug & feathery query params) or should a form have a flag indicating if it is the apex form?

File: src/integrations/stytch.ts [hunk 3]
```diff
@@ -20,7 +20,12 @@ export function installStytch(stytchConfig: any) {
       if (authState.client) resolve(authState.client);
       else {
         const initializedClient = new StytchHeadlessClient(
-          stytchConfig.metadata.token
+          stytchConfig.metadata.token,
+          {
+            cookieOptions: {
+              availableToSubdomains: true
```
Comment (obmelvin) [inline src/integrations/stytch.ts:26]: should this always be true now, or do we add a checkbox to the integration settings?

File: src/auth/LoginForm.tsx [hunk 4]
```diff
@@ -48,6 +48,7 @@ export const authState = {
 };
 
 const LoginForm = ({
+  apexForm = false,
```
Comment (bo-dun-1) [inline src/auth/LoginForm.tsx:51]: we dont need this anymore do we?

File: src/auth/LoginForm.tsx [hunk 5]
```diff
@@ -187,6 +199,10 @@ const LoginForm = ({
         <JSForm {...formProps} _internalId={_internalId} />
       </div>
     );
+  } else if (initInfo().redirectCallbacks[_internalId]) {
+    // If logged in and have finished onboarding questions for an apex form, then we need to redirect back to application
+    initInfo().redirectCallbacks[_internalId]();
+    return <></>;
```
Comment (bo-dun-1) [inline src/auth/LoginForm.tsx:205]: why return a fragment instead of null?

**Review by bo-dun-1 — APPROVED** [review]

**Pull request merged**